docs/StardustDocs/topics/operations.md

```diff
@@ -37,6 +37,14 @@ Most multiplex operations end with `into` or `with` function. The following nami
 * `into` defines column names for storing operation results. Used in [`move`](move.md), [`group`](group.md), [`split`](split.md), [`merge`](merge.md), [`gather`](gather.md), [`groupBy`](groupBy.md), [`rename`](rename.md).
 * `with` defines row-wise data transformation with [`row expression`](DataRow.md#row-expressions). Used in [`update`](update.md), [`convert`](convert.md), [`replace`](replace.md), [`pivot`](pivot.md).
 
+## List of DataRow operations
```

Comment:
Since it's a title, probably write "Operations" with a capital. Same as for the other headers in the file.

Reply:
Kotlin documentation and writerside documentation doesn't use title case, i think we probably shouldn't bother either? https://kotlinlang.org/docs/server-overview.html#deploying-kotlin-server-side-applications
https://kotlinlang.org/docs/constructing-collections.html

Reply:
Sure! But then we do need to make sure to remove all title case usages in the rest of the file(s). I'll add it [here](https://github.com/Kotlin/dataframe/issues/395#issuecomment-1587490105)